tests/unit/htdocs/class/XoopsPageNavTest.php

```
@@ -0,0 +1,662 @@
+        $this->assertSame('', $nav->renderSelect());
+    }
+
+    public function testRenderSelectReturnsEmptyWhenTotalIsZero(): void
+    {
+        $nav = new \XoopsPageNav(0, 10, 0);
+        $this->assertSame('', $nav->renderSelect());
+    }
+
+    public function testRenderSelectReturnsEmptyWhenTotalIsOne(): void
+    {
+        $nav = new \XoopsPageNav(1, 10, 0);
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '287', column '20'). (undefined)

(MissingImport)